modules/gltf/src/lib/extensions/EXT_mesh_features.ts

```diff
@@ -91,3 +93,86 @@ function processMeshPrimitiveFeatures(
     featureId.data = featureIdData;
   }
 }
+
+/*
+  Encoding data
+*/
+
+/**
+ * Ecnodes a feature ID set to extension.
+ * @param scenegraph - Instance of the class for structured access to GLTF data.
+ * @param featureIdArray - array of feature IDs.
+ * @param meshIndex - index of the mesh comntaining the target primitive.
+ * @param primitiveIndex - index of the primitive where the data encoded should be put.
+ * @returns primitive updated
+ * @see https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features
+ */
+export function encodeExtMeshFeatures(
+  scenegraph: GLTFScenegraph,
+  featureIdArray: number[] | TypedArray,
```

Comment:
Normally we would create the extension with these fields, and then call `encode()` on the extension.

Reply:
Done